components/server/ee/src/billing/billing-mode.spec.db.ts

```diff
@@ -50,24 +50,22 @@ class StripeServiceMock extends StripeService {
         super();
     }
 
-    async findUncancelledSubscriptionByCustomer(customerId: string): Promise<Stripe.Subscription | undefined> {
+    async findUncancelledSubscriptionByCustomer(customerId: string): Promise<string | undefined> {
```

Comment:
In all cases calls to this method is preceeded with getCustomer by attributionid call. Would clean the code to do this within this method. I.e. change it to `findUncancelledSubscriptionByAtrributionId`

Reply:
I'm of course also talking about the main implementation in stripe-service not only the mock :)